.ci_support/linux_64_c_compiler_version10cuda_compiler_version11.2cxx_compiler_version10python3.7.____cpython.yaml

```diff
@@ -24,8 +24,8 @@ pin_run_as_build:
     max_pin: x.x
 python:
 - 3.7.* *_cpython
-pytorch_version:
-- 1.8.0
+pytorch:
+- '1.10'
```

Comment:
Do you want to manually pull in the migrator? Otherwise, it is buliding with the old version.

Reply:
We want this to be compatible with NNPOps (https://github.com/conda-forge/nnpops-feedstock/pull/3)

Reply:
Ok, rerender if a few minutes, or in an hour:

https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/2812